xaitk_saliency/utils/tools/drise_from_config.py

```diff
@@ -0,0 +1,130 @@
+import numpy as np
+from typing import Optional, Collection
+
+from smqtk_detection.interfaces.detect_image_objects import DetectImageObjects
+from smqtk_core.configuration import from_config_dict
+
+from xaitk_saliency.impls.perturb_image.rise import RISEGrid
+from xaitk_saliency.impls.gen_detector_prop_sal.drise_scoring import DRISEScoring
+from xaitk_saliency.utils.masking import occlude_image_batch
+from xaitk_saliency.utils.detection import format_detection
+
+
+def drise_from_config(
+    ref_imgs: Collection[np.ndarray],
+    config_dict: dict,
```

Comment:
This functional component should take in an actual instance of `DetectImageObjects`. The "hydration" of the instance should probably happen during the click CLI function before calling this function.